ci-operator/config/jboss-fuse/fuse-xpaas-qe-container/jboss-fuse-fuse-xpaas-qe-container-main__jboss-fuse-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  test-image:
+    name: xpaas-qe
+    namespace: ci
+    tag: 7.12.x-openjdk1.8_local
```

Comment:
We can remove this two images, `os` and `test-image`. We are not using these for any pods in this scenario.
xpass-qe image is directly used inside step `jboss-fuse-deploy-resources`.